src/ConnectionTrait.php

```
@@ -88,40 +112,12 @@ public function setGoneAwayDetector(GoneAwayDetector $goneAwayDetector): void
      */
     private function doWithRetry(callable $callable, ?string $sql = null)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$sql'. (Unused Code Rules)

(UnusedFormalParameter)